### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by Sami32**
Correct and add documentation.

---

This change is [Reviewable](https://reviewable.io/reviews/universalmediaserver/universalmediaserver/1216)

### CODE REVIEW

File: src/main/external-resources/renderers/DefaultRenderer.conf [hunk 1]
```diff
@@ -497,11 +499,19 @@ DLNAProfileChanges =
...
+AccurateDLNAOrgPN = 
+
 # Whether to send folder thumbnails.
 # Default: true
 SendFolderThumbnails = 
 
-# Undocumented.
+# This option allows thumbnails to be generated with MediaInfo.
+# It is automatically used with unknow renderer.
+# This is only reliable when MediaInfo = true.
```
Comment (Nadahar) [inline src/main/external-resources/renderers/DefaultRenderer.conf]: "unknow**n** renderer"

File: src/main/external-resources/renderers/DefaultRenderer.conf [hunk 2]
```diff
@@ -432,19 +432,21 @@ DisableMencoderNoskip =
...
 
-# This option adds black borders to the video to prevent the renderer from
-# displaying an incorrect aspect ratio.
+# If this is true, it will always output video at 16/9 aspect ratio to the renderer,
+# meaning that all transcoded videos with different aspect ratios will have black
+# bars added to the edges to make them 16/9. It doesn't work with 3D SBS video.
 # If some videos on your renderer looked either stretched or squashed, enabling
 # this setting may fix that.
-# This is only reliable when MediaInfo = true.
 # Default: false
```
Comment (Nadahar) [inline src/main/external-resources/renderers/DefaultRenderer.conf]: Isn't that still the case?

File: src/main/external-resources/renderers/DefaultRenderer.conf [hunk 3]
```diff
@@ -432,19 +432,21 @@ DisableMencoderNoskip =
...
 SubtitleHttpHeader = 
 
-# This option adds black borders to the video to prevent the renderer from
-# displaying an incorrect aspect ratio.
+# If this is true, it will always output video at 16/9 aspect ratio to the renderer,
+# meaning that all transcoded videos with different aspect ratios will have black
+# bars added to the edges to make them 16/9. It doesn't work with 3D SBS video.
 # If some videos on your renderer looked either stretched or squashed, enabling
 # this setting may fix that.
-# This is only reliable when MediaInfo = true.
```
Comment (Nadahar) [inline src/main/external-resources/renderers/DefaultRenderer.conf]: Isn't that the case?

File: src/main/external-resources/UMS.conf [hunk 4]
```diff
@@ -1444,6 +1468,36 @@ live_subtitles_limit =
...
+# Default: false
+search_folder=
+
+# Search in folder
+# ----------------
+# Allows you to search in folders.
+# Default: false
+search_in_folder=
+
+# Search recurse
```
Comment (Nadahar) [inline src/main/external-resources/UMS.conf]: Recursive search

File: src/main/external-resources/UMS.conf [hunk 5]
```diff
@@ -1444,6 +1468,36 @@ live_subtitles_limit =
...
+# --------------
+# Allows folder recursing.
```
Comment (Nadahar) [inline src/main/external-resources/UMS.conf]: Allows recursive folder search

File: src/main/external-resources/UMS.conf [hunk 6]
```diff
@@ -454,7 +460,7 @@ fully_played_action =
 # Default: ""
 fully_played_output_directory =
 
-# Minimum watched play time
+# Minimum watched play time (non-GUI)
 # -------------------------
```
Comment (Nadahar) [inline src/main/external-resources/UMS.conf]: Why "non-GUI"?

File: src/main/external-resources/UMS.conf [hunk 7]
```diff
@@ -1306,12 +1312,30 @@ web_flash =
 # Default: false
 web_subtitles_transcoded =
 
-# Other web settings
-# ------------------
-# Not documented
+# web videos MIME types spoofing
+# ------------------------------
+# Whether Chrome browser is used, send videos with a spoofed WEBM MIME type.
```
Comment (Nadahar) [inline src/main/external-resources/UMS.conf]: *When* Chrome..

File: src/main/external-resources/UMS.conf [hunk 8]
```diff
@@ -1306,12 +1312,30 @@ web_flash =
...
+# Default: false
 web_chrome_mkv_as_webm_spoof =
+# Whether Firefox is used on Linux, send videos with a spoofed MP4 MIME type.
+# Default: false
```
Comment (Nadahar) [inline src/main/external-resources/UMS.conf]: *When* Firefox..

File: src/main/external-resources/renderers/DefaultRenderer.conf (hunk 3 above)
Comment (Sami32) [inline src/main/external-resources/renderers/DefaultRenderer.conf]: I checked, and didn't found anything that confirmed that.
That also explain why it's used with success with renderer having `MediaInfo = false`.

File: src/main/external-resources/UMS.conf (hunk 6 above)
Comment (Sami32) [inline src/main/external-resources/UMS.conf]: Where do you see it in UMS GUI ?

Comment (Nadahar) [inline src/main/external-resources/UMS.conf]: @Sami32 I haven't looked, but if we should have such a marking we should do it on all "non-GUI" settings. Is that implemented?

File: src/main/external-resources/UMS.conf [hunk 9]
```diff
@@ -231,6 +231,12 @@ renderer_force_default =
 # Default: true
 chromecast_extension =
 
+# Chromecast Debug
```
Comment (Nadahar) [inline src/main/external-resources/UMS.conf]: non-GUI?

File: src/main/external-resources/UMS.conf [hunk 10]
```diff
@@ -1306,12 +1312,30 @@ web_flash =
 # Default: false
 web_subtitles_transcoded =
 
-# Other web settings
-# ------------------
-# Not documented
+# web videos MIME types spoofing
```
Comment (Nadahar) [inline src/main/external-resources/UMS.conf]: non-GUI?

Comment (Nadahar) [inline src/main/external-resources/UMS.conf]: Web vid..

File: src/main/external-resources/UMS.conf [hunk 11]
```diff
@@ -1306,12 +1312,30 @@ web_flash =
...
+# ------------------------------
+# When Chrome browser is used, send videos with a spoofed WEBM MIME type.
```
Comment (Nadahar) [inline src/main/external-resources/UMS.conf]: non-GUI?

File: src/main/external-resources/UMS.conf [hunk 12]
```diff
@@ -1306,12 +1312,30 @@ web_flash =
...
+# Default: false
 web_chrome_mkv_as_webm_spoof =
+# When Firefox is used on Linux, send videos with a spoofed MP4 MIME type.
```
Comment (Nadahar) [inline src/main/external-resources/UMS.conf]: non-GUI?

File: src/main/external-resources/UMS.conf [hunk 13]
```diff
@@ -1444,6 +1468,36 @@ live_subtitles_limit =
 # Default: 0 (meaning keep indefinitely)
 live_subtitles_timeout =
 
+################################################
+## Server side search in web interface (Non GUI)
+################################################
+
+# Search folder
```
Comment (Nadahar) [inline src/main/external-resources/UMS.conf]: non-GUI?

File: src/main/external-resources/UMS.conf [hunk 14]
```diff
@@ -1444,6 +1468,36 @@ live_subtitles_limit =
 # Default: 0 (meaning keep indefinitely)
 live_subtitles_timeout =
 
+################################################
+## Server side search in web interface (Non GUI)
```
Comment (Nadahar) [inline src/main/external-resources/UMS.conf]: **n**on**-**GUI?

File: src/main/external-resources/UMS.conf [hunk 15]
```diff
@@ -1444,6 +1468,36 @@ live_subtitles_limit =
...
+
+# Search folder
+#--------------
+# Allows you to send a search string when requesting a folder. If enabled you
+# should see a folder called "Search Disc Folders" in your root folder which
+# accept search strings and searches your shared folder.
+# Default: false
+search_folder=
+
+# Search in folder
```
Comment (Nadahar) [inline src/main/external-resources/UMS.conf]: non-GUI?

File: src/main/external-resources/UMS.conf [hunk 16]
```diff
@@ -1444,6 +1468,36 @@ live_subtitles_limit =
...
+# ----------------
+# Allows you to search in folders.
+# Default: false
+search_in_folder=
+
+# Recursive search
```
Comment (Nadahar) [inline src/main/external-resources/UMS.conf]: non-GUI?

File: src/main/external-resources/UMS.conf [hunk 17]
```diff
@@ -1444,6 +1468,36 @@ live_subtitles_limit =
...
+# ----------------
+# Allows recursive folder search.
+# Default: true
+search_recurse=
+
+# Recursive search depth
```
Comment (Nadahar) [inline src/main/external-resources/UMS.conf]: non-GUI?

File: src/main/external-resources/renderers/DefaultRenderer.conf (hunk 3 above)
Comment (Nadahar) [inline src/main/external-resources/renderers/DefaultRenderer.conf]: Ok, I'm guessing the idea was that only MediaInfo could properly parse the resolution, but I'd think FFmpeg does that just fine as well.

File: src/main/external-resources/renderers/DefaultRenderer.conf [hunk 18]
```diff
@@ -497,11 +499,19 @@ DLNAProfileChanges =
 # Default: true
 DLNAOrgPN = 
 
+# Spends a longer time analyzing the contents of files to determine the most accurate information to send renderers.
```
Comment (Nadahar) [inline src/main/external-resources/renderers/DefaultRenderer.conf]: non-GUI?

File: src/main/external-resources/renderers/DefaultRenderer.conf [hunk 19]
```diff
@@ -497,11 +499,19 @@ DLNAProfileChanges =
...
+# Warning: This can cause significantly slower browse times.
+# Default: false
+AccurateDLNAOrgPN = 
+
 # Whether to send folder thumbnails.
 # Default: true
 SendFolderThumbnails = 
 
-# Undocumented.
+# This option allows thumbnails to be generated with MediaInfo.
```
Comment (Nadahar) [inline src/main/external-resources/renderers/DefaultRenderer.conf]: non-GUI?

File: src/main/external-resources/UMS.conf (hunk 6 above)
Comment (Sami32) [inline src/main/external-resources/UMS.conf]: Yes.

File: src/main/external-resources/UMS.conf (hunk 9 above)
Comment (Sami32) [inline src/main/external-resources/UMS.conf]: Yes ;)

File: src/main/external-resources/UMS.conf (hunk 6 above)
Comment (Nadahar) [inline src/main/external-resources/UMS.conf]: Ok, I've never seen that.

File: src/main/external-resources/UMS.conf (hunk 10 above)
Comment (Sami32) [inline src/main/external-resources/UMS.conf]: All the undocumented are non-GUI, but as the old ones wasn't marked as it, i just did the same.

File: src/main/external-resources/renderers/DefaultRenderer.conf (hunk 3 above)
Comment (Sami32) [inline src/main/external-resources/renderers/DefaultRenderer.conf]: It's to circumvent the "buggy" renderers, as some Panasonic and Sony, that streches/"compress" any videos (4/3, 21/10...) to 16/9

Comment (Nadahar) [inline src/main/external-resources/renderers/DefaultRenderer.conf]: @Sami32 I understand that part, what I was referring to was why it was claimed to only be reliable using MediaInfo. The only reason I can see for this claim is if only MediaInfo can reliably parse the resolution.

File: src/main/external-resources/UMS.conf (hunk 15 above)
Comment (Sami32) [inline src/main/external-resources/UMS.conf]: It's already said above in the general section ===================

File: src/main/external-resources/renderers/DefaultRenderer.conf (hunk 18 above)
Comment (Sami32) [inline src/main/external-resources/renderers/DefaultRenderer.conf]: No.

Comment (Sami32) [inline src/main/external-resources/renderers/DefaultRenderer.conf]: Yes ;)

**Comment by Sami32** [comment]
@Nadahar Thank you for your approval, iknow that it costed you a lot...

BTW, sorry, as i just saw that all my reviews was pending since then, so i'm not sure that i should do that now ? I guess not as you could change your mind, i better resolve, rebase and merge it fastly :stuck_out_tongue_winking_eye: 
It's the second time that i see that i forget to validate my review. Alzeimer is coming ;)

**Comment by Nadahar** [comment]
@Sami32 It didn't cost me a lot, I didn't find anything wrong here. I just forget to do this among all the other stuff that come up.

I'm not sure what you mean by "all my reviews was pending since then". Do you mean in this PR? Have you reviewed this?

**Comment by Sami32** [comment]
When i re-checked at it after being very surprise by your approval, i just saw "pending" on all my reviews, answering your previous comment 23 days ago...

**Comment by Nadahar** [comment]
@Sami32 Ok, I see. I always submit replies at once, because I'm senile too...

**Pull request merged**